lib/posix/options/Kconfig.deprecated

```diff
@@ -56,6 +56,14 @@ config POSIX_MQUEUE
 
 	  Please use CONFIG_POSIX_RW_LOCKS instead.
 
+config POSIX_SYSLOG
+	bool "Support for syslog() [DEPRECATED]"
+	select DEPRECATED
+	help
+	  This option is deprecated.
+
+	  Please use CONFIG_XSI_SYSTEM_LOGGING instead.
```

Comment:
`XSI_SYSTEM_LOGGING` isnt defined in this commit

Reply:
I think I may have lost it in a rebase -- working on it..

Reply:
Fixed!